src/modules/snowflake/data-connector/lambda_connectors/attribute_property_value_reader_by_entity.py

```diff
@@ -0,0 +1,80 @@
+import logging
```

Comment:
there's some header content we should include in all files regarding the licensing. e.g. as a reference: https://github.com/aws-samples/aws-iot-twinmaker-samples/blob/main/src/libs/udq_helper_utils/udq_utils/udq.py#L1

Reply:
Will fix in next commit